packages/use-contractkit/src/modals/styles.ts

```diff
@@ -12,3 +12,18 @@ export const defaultModalStyles = {
     boxShadow: '0px 3px 4px 0px rgba(0, 0, 0, 0.15)',
   },
 };
+
+export const mobileModalStyles = {
```

Comment:
Can this be a css file instead? We already have a precedent for that elsewhere in use-ck.